src/Edge.ts

```diff
@@ -188,6 +188,23 @@ export class Edge {
         return this.issues();
     }
 
+    getIconColor : ko.PureComputed<string> = ko.pureComputed(() => {
+        const errorsWarnings = this.getErrorsWarnings()
+
+        if(errorsWarnings.errors.length>0){
+            return '#ea2727'
```

Comment:
These colour literals are used in Node, Edge and Field.

Can we add them once to EagleConfig.ts and then request them by name from here?